content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md

```diff
@@ -445,8 +445,8 @@ This guide demonstrates how to get coding suggestions from {% data variables.pro
 {% data variables.product.prodname_copilot %} offers coding suggestions as you type. For example, type this function
 signature in a Swift file:
 
-```shell copy
-func CalculateDaysBetweenDates(
+```swift copy
```

Comment:
Now I see why you didn't use Swift. I'll check why that's not included in our supported languages.

Reply:
I've opened:

- https://github.com/github/docs/pull/35732

Once that's approved and merged, we can update your branch and merge this.

Reply:
That looks better 🙂 

<img width="749" alt="Screenshot showing the updated content in the preview." src="https://github.com/user-attachments/assets/bebb60a0-37ff-4dba-9590-569b6bb75002" />